conduit/services/articles.service.js

```diff
@@ -187,22 +187,6 @@ module.exports = {
 				if (ctx.params.tag)
 					params.query.tagList = { "$in" : [ctx.params.tag] };
 
-				if (ctx.params.author) {
```

Comment:
Please put back these parts just commenting out.

Reply:
Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22